test/world_database_test.rb

```diff
@@ -93,7 +89,7 @@ def test_it_can_have_data_inserted
 
     assert File.exists?('./db/database_name.db')
 
-    sqldb = SQLite3::Database.new("./db/database_name.db")
+    sqldb = wdb.file
```

Comment:
I think the reason I did it the way I did (instead of using the wdb) is because it makes the test independent of the code we are testing.  

By changing this to use the WorldDatabase method instead of the SQLite3::Database.new("./db/database_name.db") we are relying on code that might not be written for our test setup.

Generally (I took this for granted until I reviewed the code again) we want our test setups to be independent of the code we are testing.  That way it is very clear what is being tested and how we are setting it up.  

For example, if we changed the 'file' method to be named 'database' we would need to refactor a lot of our tests here.  I don't think this is an absolute rule, so it is entirely reasonable to make a different decision here.  I recently had to refactor a lot of tests based on a single method refactor, so I think it is something good to think about when writing tests to save you work later down the road.